akka-docs/src/main/paradox/discovery/index.md

```diff
@@ -219,7 +218,7 @@ akka {
   discovery {
     method = aggregate
     aggregate {
-      discovery-methods = ["akka-dns", "config"]
+      discovery-methods = ["config", "akka-dns"]
```

Comment:
I know the ordering is debatable, but isn't that so that it's preferable to lookup static configs and only do network lookup when needed? 

As previously discussed, I don't think that we should consider the mix of dev/prod configs. This is a problem of another sort and users must have different config files instead of relying on ordering to correctly select things in dev/prod.

Anyway, this is just a suggestion.